ios/.gitignore

```diff
@@ -0,0 +1,90 @@
+# Xcode
```

Comment:
Add .DS_Store files to gitignore + remove the existing ones.

Reply:
Done!